messages/agent.test.run.md

```diff
@@ -12,7 +12,7 @@ By default, this command outputs test results in human-readable tables for each
 
 # flags.api-name.summary
 
-API Name of the agent test to start.
+API name of the AiEvaluationDefinition to run.
```

Comment:
Hmmm, I've been trying to keep the metadata type names out of the help as much as possible, because users don't necessary know what "AIEvalutationDefinition" means.  so maybe we compromise and say:

API name of the agent test to run; corresponds to the name of the AiEvaluationDefinition metadata component that implements the agent test.